jubatus/core/clustering/clustering_config.hpp

```diff
@@ -72,7 +78,9 @@ struct clustering_config {
         & JUBA_MEMBER(compressed_bucket_size)
         & JUBA_MEMBER(forgetting_factor)
         & JUBA_MEMBER(forgetting_threshold)
-        & JUBA_MEMBER(seed);
+        & JUBA_MEMBER(eps)
+        & JUBA_MEMBER(seed)
+        & JUBA_MEMBER(min_core_point);
```

Comment:
@yukimori @TkrUdagawa 
This comment is important for usability.
Do you plan to fix this point for 1.0 release?